butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -143,10 +149,18 @@
     debuggable = !"false".equals(env.getOptions().get(OPTION_DEBUGGABLE));
 
     typeUtils = env.getTypeUtils();
+    elementUtils = env.getElementUtils();
     filer = env.getFiler();
     try {
-      trees = Trees.instance(processingEnv);
-    } catch (IllegalArgumentException ignored) {
+      //reflection won't be necessary after https://github.com/gradle/gradle/pull/8393
```

Comment:
The issue is closed, is the comment still relevant?

Reply:
I think it still provides some context. Let me know if you want me to remove it.

Reply:
If the issue isn't getting fixed, maybe we can change it. What would future dev would like to see here?